public/walkthroughs/1/walkthrough.adoc

```diff
@@ -114,41 +114,40 @@ To receive messages from EnMasse, you create a connection in Red Hat Fuse Online
 
 . Select the *Create Connection* button to start the *Create Connection* wizard.
 
-. Select *AMQP* to configure an *AMQP Message Broker* connection.
+. When prompted with *Select Connector*, select *AMQ Message Broker*.
 
-. Enter the connection URI relating to {enmasse}:
+. When prompted with *AMQ Message Broker Configuration*:
+.. Enter the following into the *Broker URL* field:
 +
 [subs="attributes+"]
 ----  
-amqp://{amq-broker-url}:5672?amqp.saslMechanisms=PLAIN
+tcp://{messaging-broker-url}:61616
```

Comment:
@finp @aidenkeating  hmm, according to the list of attributes it should be `amp-url`: https://github.com/integr8ly/tutorial-web-app/blob/add-own-walkthroughs/public/walkthroughs/my-custom-walkthrough/walkthrough.adoc#using-walkthrough-attributes

Reply:
@pb82  I think`{amq-url}` is the URL for the AMQ console, this requires the AMQ broker URL. They could probably be named better though

Reply:
@pb82 using {amq-broker-url}